exp/clustering.md

```diff
@@ -1,20 +1,38 @@
-While it is possible to install Anbox Cloud on a single machine, you usually want to distribute the load over several machines in a cluster. In such a clustering setup, one node is dedicated to host the AMS service (the management node). If you use the streaming stack, two additional nodes are dedicated to host the extra services required for streaming. All other nodes are used as worker nodes.
+While it is possible to install Anbox Cloud on a single machine, you usually want to distribute the load over several machines in a cluster.
 
-Each worker node runs [LXD](https://linuxcontainers.org/) in clustering mode, and this LXD cluster is used to host the Android containers.
+The Anbox Cloud Appliance provides basic and experimental support for multi-node setups. See [Multi-node support for the Anbox Cloud Appliance](#multi-node-appliance).
+
+If you're running Anbox Cloud, full clustering support is available. See [Clustering for full Anbox Cloud deployments](#clustering-anbox-cloud).
+
+<a name="multi-node-appliance"></a>
+## Multi-node support for the Anbox Cloud Appliance
+
+If you're running the Anbox Cloud Appliance and you want to extend the available container capacity, you can join additional machines to the appliance.
+
+The appliance uses [FAN networking](https://wiki.ubuntu.com/FanNetworking) to spawn an overlay network that allows containers on different machines to communicate with each other.
```

Comment:
We need to emphasize here a bit more that FAN networking is a strict requirement and that FAN networking requires a /16 or /24 subnet on the NIC and otherwise doesn't work. We detect during `anbox-cloud-appliance init` what subnet the user is on and disable FAN networking in case the subnet isn't a /16 or /24.